packages/http-client-java/generator/http-client-generator/src/main/java/com/microsoft/typespec/http/client/generator/model/EmitterOptions.java

```diff
@@ -22,7 +22,7 @@ public class EmitterOptions implements JsonSerializable<EmitterOptions> {
     private List<String> serviceVersions;
     private Boolean generateTests = true;
     private Boolean generateSamples = true;
-    private Boolean enableSyncStack;
+    private Boolean enableSyncStack = true;
```

Comment:
Just to confirm: the description states "enable sync-stack for TypeSpec management by default." Is the default value we set here still limited to management, or is it intended to be enabled for all?

Reply:
I think data-plane is already `true` before this PR (though still settable via option).

Reply:
> Just to confirm: the description states "enable sync-stack for TypeSpec management by default." Is the default value we set here still limited to management, or is it intended to be enabled for all?

Previously DPG TypeSpec is already on sync-stack. This PR is to enable for MPG TypeSpec as well.